connectivity/drivers/emac/TARGET_STM/TARGET_STM32H7/TARGET_PORTENTA_H7/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+target_sources(mbed-emac
```

Comment:
As the content is too small, this can be added in the list file that add this one to the build.
This list file can then just be deleted.

Reply:
The same applies to all files in that subdirectory, like `connectivity/drivers/emac/TARGET_STM/TARGET_STM32H7/TARGET_NUCLEO_H743ZI/CMakeLists.txt`. Can we do it with an atomic patch after the merge?